include/wx/qt/dataobj.h

```diff
@@ -12,13 +12,20 @@
 // wxDataObject is the same as wxDataObjectBase under wxQT
 // ----------------------------------------------------------------------------
 
+class QMimeData;
+
 class WXDLLIMPEXP_CORE wxDataObject : public wxDataObjectBase
 {
 public:
     wxDataObject();
     virtual ~wxDataObject();
 
     virtual bool IsSupportedFormat( const wxDataFormat& format, Direction dir = Get ) const;
+	virtual void AddDataTo(QMimeData &mimeData) const;
```

Comment:
All these new functions should have `Qt` prefix to indicate that they're wxQt-specific.